File: dotnet/Multiplayer/Matchmaking/MatchmakingKickRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Michitai.Multiplayer.Matchmaking
{
    /// <summary>
    /// Request data for kicking a player from a matchmaking lobby.
    /// </summary>
    public class MatchmakingKickRequest
    {
        /// <summary>
        /// The ID of the player to kick.
        /// </summary>
        public int Player_id { get; set; }
    }
}

File: java/src/main/java/com/michitai/multiplayer/matchmaking/MatchmakingKickRequest.java
package com.michitai.multiplayer.matchmaking;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Request for kicking a player from a matchmaking lobby.
 */
public class MatchmakingKickRequest {
    @JsonProperty("player_id")
    private int playerId;

    public MatchmakingKickRequest(int playerId) {
        this.playerId = playerId;
    }

    public int getPlayerId() {
        return playerId;
    }

    public void setPlayerId(int playerId) {
        this.playerId = playerId;
    }
}
